### PR TITLE
PMM-7313 RD-PostgreSQL

### PR DESCRIPTION
**Pull request by percona-csalguero**
[PMM-7313 - Amazon RDS PostgreSQL monitoring](https://jira.percona.com/browse/PMM-7313)

- [x] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1555

---
- [X] Tests passed.
- [X] Feature build pass.
- [ ] (Re)requested review.
- [X] Update API dependency.

### CODE REVIEW

File: services/management/rds.go [hunk 1]
```diff
@@ -72,8 +72,8 @@ var (
 		"mysql":        managementpb.DiscoverRDSEngine_DISCOVER_RDS_MYSQL,
 
 		// TODO https://jira.percona.com/browse/PMM-5195
```
Comment (BupycHuk) [inline services/management/rds.go:74]: let's remove this TODO

File: services/management/rds.go [hunk 2]
```diff
@@ -367,6 +367,82 @@ func (s *RDSService) AddRDS(ctx context.Context, req *managementpb.AddRDSRequest
...
+			if err != nil {
+				return err
+			}
+			invService, err := services.ToAPIService(service)
+			if err != nil {
+				return err
+			}
+			res.Postgresql = invService.(*inventorypb.PostgreSQLService)
+
+			_, err = supportedMetricsMode(tx.Querier, req.MetricsMode, models.PMMServerAgentID)
```
Comment (BupycHuk) [inline services/management/rds.go:393]: it's always should be pull metrics mode for pmm-server agent

**Review by BupycHuk — COMMENTED** [review]
overall looks good to me, just a few minor comments

**Review by JiriCtvrtka — APPROVED** [review]
LGTM, just please remove comments:
https://github.com/percona/pmm-managed/pull/681/files#diff-369d0523ffa337eded57c08518d3f26daee222238cee512568eabfa4e18370ddR355

https://github.com/percona/pmm-managed/pull/681/files#diff-369d0523ffa337eded57c08518d3f26daee222238cee512568eabfa4e18370ddR362

**Review by askomorokhov — APPROVED** [review]

File: services/management/rds.go [hunk 3]
```diff
@@ -82,8 +82,8 @@ var (
 		pointer.ToString("mysql"),
 
 		// TODO https://jira.percona.com/browse/PMM-5195
```
[the remark this answers is not in the document]
Comment (percona-csalguero) [reply services/management/rds.go]: done

File: services/management/rds.go (hunk 1 above)
Comment (percona-csalguero) [reply services/management/rds.go]: done

File: services/management/rds_test.go [hunk 4]
```diff
@@ -280,4 +280,88 @@ func TestRDSService(t *testing.T) {
...
+				CustomLabels: map[string]string{
+					"foo": "bar",
+				},
+			},
+			PostgresqlExporter: &inventorypb.PostgresExporter{
+				AgentId:    "/agent_id/00000000-0000-4000-8000-00000000000d",
+				PmmAgentId: "pmm-server",
+				ServiceId:  "/service_id/00000000-0000-4000-8000-00000000000c",
+				Username:   "username",
+				//TablestatsGroupTableLimit: 1000,
```
[the remark this answers is not in the document]
Comment (percona-csalguero) [reply services/management/rds_test.go]: done

File: services/management/rds.go [hunk 5]
```diff
@@ -367,6 +365,82 @@ func (s *RDSService) AddRDS(ctx context.Context, req *managementpb.AddRDSRequest
...
+			if err != nil {
+				return err
+			}
+			invService, err := services.ToAPIService(service)
+			if err != nil {
+				return err
+			}
+			res.Postgresql = invService.(*inventorypb.PostgreSQLService)
+
+			_, err = supportedMetricsMode(tx.Querier, managementpb.MetricsMode_PULL, models.PMMServerAgentID)
```
Comment (BupycHuk) [inline services/management/rds.go:391]: let's revert my last comment related to this line.

Comment (percona-csalguero) [reply services/management/rds.go:391]: which comment?

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**